cmd/notation/inspect.go

```diff
@@ -248,21 +242,34 @@ func getSignedAttributes(outputFormat string, envContent *signature.EnvelopeCont
 	return signedAttributes
 }
 
-func getUnsignedAttributes(outputFormat string, envContent *signature.EnvelopeContent) (map[string]any, error) {
-	unsignedAttributes := map[string]any{}
+func getUnsignedAttributes(outputFormat string, envContent *signature.EnvelopeContent) map[string]any {
+	unsignedAttributes := make(map[string]any)
+
+	if envContent.SignerInfo.UnsignedAttributes.SigningAgent != "" {
+		unsignedAttributes["signingAgent"] = envContent.SignerInfo.UnsignedAttributes.SigningAgent
+	}
 
 	if envContent.SignerInfo.UnsignedAttributes.TimestampSignature != nil {
```

Comment:
It is better to refactor this if statement block in case we will have more unsigned attributes in the future. Note the `return` statement will do a hard cut.

Reply:
Updated.